index.html

```diff
@@ -118,15 +209,31 @@
     // TRANSFORM
     txData = data.slice();
 
-    if (options.yearRange !== 'all') {
-      var yearRange = options.yearRange.split('-');
+    if (options.SOCGroup !== 'all') {
+      var SOCGroup=options.SOCGroup;
```

Comment:
A purely stylistic note, Javascript style conventions frown on starting variables with capital letters. This generally indicates a CONSTANT, something like we used in Week 2 to declare a magic number at the top of our code, or a Constructor, something that we'll cover next week.

Also, note that we could just use `options.SOCGroup` directly on line 215. The only reason I created a variable at the top of this block was to use `string.split()` to divide the year range string into an array.